it/repositories/ThrottlingRepositorySpec.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.scalatest.{BeforeAndAfterEach, OptionValues}
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.inject
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.{JsObject, Json}
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+
+import scala.concurrent.ExecutionContext.Implicits._
+import scala.concurrent.Future
+
+class ThrottlingRepositorySpec extends AnyWordSpecLike with Matchers with OptionValues with GuiceOneAppPerSuite with BeforeAndAfterEach {
+
+  val testThrottleIdOne: String = "testThrottleIdOne"
+  val testThrottleIdTwo: String = "testThrottleIdTwo"
+
+  val testThrottleInstantNow: Long = 1000000
```

Comment:
Again, a fuzzy value might be better?

Reply:
Generate a random string.  Often a good idea when doing anything vaguely near persistence to make sure that old values aren't hanging around making tests appear to pass.